components/nimbus/android/src/main/java/org/mozilla/experiments/nimbus/FeatureVariables.kt

```diff
@@ -126,7 +126,7 @@ interface VariablesWithContext : Variables {
     // defaults from manifest information.
     fun asText(res: Int) = context.getString(res)
     fun asDrawable(res: Int) = context.getDrawable(res)
-    fun asText(string: String) = asStringResource(string)?.let(this::asText) ?: string
+    fun asText(string: String): String? = asStringResource(string)?.let(this::asText) ?: string
```

Comment:
This was the magic sauce necessary to avoid a recursion error in Kotlin's type resolution.